src/web/util.js

```diff
@@ -11,3 +13,14 @@ export const hashCode = str => {
   }
   return hash
 }
+
+export const parseQueryString = queryString => {
+  const params = {}
+  const queries = queryString.split('&')
+
+  for (let i = 0, l = queries.length; i < l; i++) {
+    const temp = queries[i].split('=')
+    params[temp[0]] = temp[1]
```

Comment:
depending on the user-case you may also want to `decodeURIComponent` on the value

Reply:
Just moved that logic here. But OK, I've wrapped it in `decodeURIComponent` call. )